### DISCUSSION TITLE
Esp32 +8 relay losing settings and connection to wifi

### DISCUSSION THREAD

**Discussion by gbananachewicz** [arendst/Tasmota#15971] [Q&A] [answered]
Hello, I have a board with 8 relays and an Esp32 wrooom 32d, a 5v power supply for the breadboard (5 / 3.3v) with a USB port and a 12v plug.

 removed link

 removed link

 The problem is that he loses settings and blinks quickly.  Then there is no connection to wifi and settings.  I have tried various GPIOs.  At the moment I am using GPIO D13, D12, D14, D27, D33, D32, D15 & D4.

 It does not issue a hotspot, you can connect with a USB cable and use the tasmota installer on the tasmota instal website to "connect to wifi".

**Comment by sfromis** [comment, marked as answer]
Sounds like a boot loop "saved" by the Tasmota automatic recovery feature of "scaling back" config till able to boot without crashing again. Such can have many causes, including dubious power available.
